coldfront/core/group/tasks.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth.models import Group
+from coldfront.core.user.models import User
+from coldfront.core.project.models import (
+    Project,
+    ProjectUser,
+    ProjectUserRoleChoice,
+    ProjectUserStatusChoice,
+)
+
+
+def grant_usersupport_global_project_manager() -> None:
+    group_name = "RIS-UserSupport"
+    group = Group.objects.filter(name=group_name).first()
+
+    if not group:
+        return
+
+    all_projects = Project.objects.all()
+    group_users = User.objects.filter(groups=group)
+    role_choice = ProjectUserRoleChoice.objects.filter(name="Manager").first()
+    status_choice = ProjectUserStatusChoice.objects.filter(name="Active").first()
+
+    if not role_choice or not status_choice:
+        return
+
+    # Iterate over all projects
+    for project in all_projects:
+        project_users = ProjectUser.objects.filter(
+            project=project, user__in=group_users
+        )
+        existing_project_users = {pu.user_id: pu for pu in project_users}
```

Comment:
@gunnarwustl how many users are we expected to create or update project users? If the number is in the thousands, I can see the benefit of bulk creates or updates; otherwise, invoking save or create during every iteration is not much of a penalty for small result sets. If a large set or long transactions an issue, did you consider ORM transactions?

Reply:
I'm only making one group here. 
In the future I'm sure we'll end up making more.

Reply:
Dismissing this for now so that I can move onto on-call responsibilities. The change to use of transactions is definitely recommended, especially as we add more users later.